### ISSUE TITLE
Building failure

### ISSUE THREAD

**Issue by NeonKnightOA** [NeonKnightOA/oagamepack#5]
Leaving this output from Linux as of the latest commit:

```
Traceback (most recent call last):
  File "./create_ent.py", line 49, in <module>
    entities = readCsvToDicts("entities.csv","item")
  File "./create_ent.py", line 40, in readCsvToDicts
    return readCsvToDictsAppend(result,filename,keyname)
  File "./create_ent.py", line 31, in readCsvToDictsAppend
    rowDict = row._asdict()
AttributeError: 'tuple' object has no attribute '_asdict'
Traceback (most recent call last):
  File "./create_def.py", line 46, in <module>
    entities = readCsvToDicts("entities.csv","item")
  File "./create_def.py", line 40, in readCsvToDicts
    return readCsvToDictsAppend(result,filename,keyname)
  File "./create_def.py", line 31, in readCsvToDictsAppend
    rowDict = row._asdict()
AttributeError: 'tuple' object has no attribute '_asdict'
Traceback (most recent call last):
  File "./create_ent.py", line 49, in <module>
    entities = readCsvToDicts("entities.csv","item")
  File "./create_ent.py", line 40, in readCsvToDicts
    return readCsvToDictsAppend(result,filename,keyname)
  File "./create_ent.py", line 31, in readCsvToDictsAppend
    rowDict = row._asdict()
AttributeError: 'tuple' object has no attribute '_asdict'
Traceback (most recent call last):
  File "./create_def.py", line 46, in <module>
    entities = readCsvToDicts("entities.csv","item")
  File "./create_def.py", line 40, in readCsvToDicts
    return readCsvToDictsAppend(result,filename,keyname)
  File "./create_def.py", line 31, in readCsvToDictsAppend
    rowDict = row._asdict()
AttributeError: 'tuple' object has no attribute '_asdict'
```

Tried finding a solution on the 'net, to no avail.

**Comment by sago007** [comment]
My search suggested that it is a bug fixed in python3 3.5.2. I can run them on 3.6.7

**Comment by sago007** [comment]
I believe this is solved by e58b27641ae3a2e3fdfbaea5f9ab2d086684220b
Tested on a docker image with Mint 17.3

**Comment by NeonKnightOA** [comment]
Solved, indeed. Thanks!
